image-annotator/src/redux/selectors.js

```diff
@@ -0,0 +1,9 @@
+export const selectedFace = state => state.faces.list[state.faces.selected];
```

Comment:
These are pretty important to test also IMO -- feel free to make just a `mockState` object to test them with.